src/contracts/LidoARM.sol

```diff
@@ -0,0 +1,117 @@
+ * It also integrates to a CapManager contract that caps the amount of assets a liquidity provider
+ * can deposit and caps the ARM's total assets.
+ * A performance fee is also collected on increases in the ARM's total assets.
+ * @author Origin Protocol Inc
+ */
+contract LidoARM is Initializable, AbstractARM {
+    /// @notice The address of the Lido stETH token
+    IERC20 public immutable steth;
+    /// @notice The address of the Wrapped ETH (WETH) token
+    IWETH public immutable weth;
+    /// @notice The address of the Lido Withdrawal Queue contract
+    IStETHWithdrawal public immutable lidoWithdrawalQueue;
+
+    /// @notice The amount of stETH in the Lido Withdrawal Queue
+    uint256 public lidoWithdrawalQueueAmount;
+
+    /// @param _steth The address of the stETH token
+    /// @param _weth The address of the WETH token
+    /// @param _lidoWithdrawalQueue The address of the Lido's withdrawal queue contract
+    /// @param _claimDelay The delay in seconds before a user can claim a redeem from the request
+    constructor(address _steth, address _weth, address _lidoWithdrawalQueue, uint256 _claimDelay)
+        AbstractARM(_weth, _steth, _weth, _claimDelay)
+    {
+        steth = IERC20(_steth);
+        weth = IWETH(_weth);
+        lidoWithdrawalQueue = IStETHWithdrawal(_lidoWithdrawalQueue);
+    }
+
+    /// @notice Initialize the storage variables stored in the proxy contract.
+    /// The deployer that calls initialize has to approve the this ARM's proxy contract to transfer 1e12 WETH.
+    /// @param _name The name of the liquidity provider (LP) token.
+    /// @param _symbol The symbol of the liquidity provider (LP) token.
+    /// @param _operator The address of the account that can request and claim Lido withdrawals.
+    /// @param _fee The performance fee that is collected by the feeCollector measured in basis points (1/100th of a percent).
+    /// 10,000 = 100% performance fee
+    /// 1,500 = 15% performance fee
+    /// @param _feeCollector The account that can collect the performance fee
+    /// @param _capManager The address of the CapManager contract
+    function initialize(
+        string calldata _name,
+        string calldata _symbol,
+        address _operator,
+        uint256 _fee,
+        address _feeCollector,
+        address _capManager
+    ) external initializer {
+        _initARM(_operator, _name, _symbol, _fee, _feeCollector, _capManager);
+
+        // Approve the Lido withdrawal queue contract. Used for redemption requests.
+        steth.approve(address(lidoWithdrawalQueue), type(uint256).max);
+    }
+
+    /**
+     * @notice Request a stETH for ETH withdrawal.
+     * Reference: https://docs.lido.fi/contracts/withdrawal-queue-erc721/
+     * Note: There is a 1k amount limit. Caller should split large withdrawals in chunks of less or equal to 1k each.)
+     */
+    function requestLidoWithdrawals(uint256[] memory amounts)
```

Comment:
🟠 We should emit some kind of event on `requestLidoWithdrawals` and `claimLidoWithdrawals`

Reply:
I've added `RequestLidoWithdrawals` and `ClaimLidoWithdrawals` events